script/list-page.js

```diff
@@ -0,0 +1,205 @@
+const sidebar = document.querySelector("#sidebar-product");
+const listpage = document.querySelector("#list");
+const gridpage = document.querySelector("#grid");
+const show = document.querySelector("#show");
+const paginationID = document.querySelector("#pagination");
```

Comment:
```suggestion
const pagination = document.querySelector("#pagination");
```
Như này hoặc `paginationEl` thôi là được rồi em nhé, vì nó trỏ tới html element mà